### PR TITLE
Add iterations

### PR DESCRIPTION
**Pull request by AliSchlereth**
@VictoriaVasys @mikedao 
Added iterations. 1-3 there isn't much to the directions, did I miss anything? Does 4 make sense? Did I give to much information in the suggested output?

### CODE REVIEW

File: module1/homework/bad_connection.md [hunk 1]
```diff
@@ -61,3 +61,38 @@ In your code you'll definitely need to use `if` and likely an `elsif` and `else`
...
+Refactor your program to use methods while maintaining above functionality  
+
+## Iteration 3 
+
+Refactor your program to use Classes as well as methods while maintaining above functionality  
+
+## Iteration 4  
+
+Add the following functionality to your program. 
+When I input `print log`, it prints out a summary of the conversation noting the input correlated to output and a count of how many times an output has be given.
```
Comment (VictoriaVasys) [inline module1/homework/bad_connection.md]: Do you mean how many times an input was given? Isn't the count counting how many times you hit enter, asking questions?

File: module1/homework/bad_connection.md [hunk 2]
```diff
@@ -61,3 +61,38 @@ In your code you'll definitely need to use `if` and likely an `elsif` and `else`
...
+
+## Iteration 4  
+
+Add the following functionality to your program. 
+When I input `print log`, it prints out a summary of the conversation noting the input correlated to output and a count of how many times an output has be given.   
+This may look something like this: 
+```
+HELLO?!
+* ['', '', '']
+* 3 times
```
Comment (VictoriaVasys) [inline module1/homework/bad_connection.md]: I would just add a descriptive end to these, like `3 statements were made before a response was given` or something

**Comment by mikedao** [comment]
Looking great!

File: module1/homework/bad_connection.md (hunk 1 above)
Comment (VictoriaVasys) [reply module1/homework/bad_connection.md]: Also, maybe clarify 'When I input `print log` into the CLI while bad_connection is running'; I was a little lost on where you were inputting that, but might just be me.

**Pull request merged**